base/applications/mscutils/eventvwr/lang/zh-HK.rc

```diff
@@ -172,13 +175,13 @@ BEGIN
 /* Please note that explicit TAB characters '\t' are used to fix the alignment of the message in the MessageBox */
     IDS_USAGE "ReactOS Event Viewer\n\
 \n\
-EventVwr [電腦名稱] [/l:<事件記錄檔案>] [/?]\n\
+EventVwr [電腦名稱] [/L:<事件記錄檔案>] [/?]\n\
```

Comment:
Please check tr-TR.rc it needs a similar adjustment.

Reply:
Resolved.